rcsb/utils/repository/RepositoryProvider.py

```diff
@@ -574,8 +573,8 @@ def __getEntryUriList(self, idCodeList=None, mergeContentTypes=None):
                 if mergeContentTypes and "vrpt" in mergeContentTypes:
                     # if self.__chP.hasEntryContentType(tId, "validation_report"):
                     if self.__chP.hasValidationReportData(tId):
-                        kwD = HashableDict({"marshalHelper": toCifWrapper})
-                        locObj.append(HashableDict({"locator": self.__getLocatorRemote("validation_report", tId), "fmt": "xml", "kwargs": kwD}))
+                        kwD = HashableDict({})
```

Comment:
Same question here (but in reference to line 571).

Reply:
And actually, for both questions, now that you're not changing `kwD` between consecutive loop iterations, this could just be defined once outside the scope of the loop?